### PR TITLE
minor resourcecheck old code cleanup

### PR DESCRIPTION
**Pull request by igorkis-scrts**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Seems to be that `fn_resourcecheck` uses leftover Barbolani's code that no longer exists/used anymore (`statusAct` nowhere to be found in the current state of codebase) - seems that at some point there were armed CIVILIAN-side, uh, civilians, that required special cleanup on each resource tick. 
So, this PR replaces this code with simple post-mortem call which is used everywhere in same cases.  Also, it will make resource tick a little bit faster under certain circumstances as it will no longer iterate through `allUnits`.

### Please specify which Issue this PR Resolves.
-

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [x] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
-

### CODE REVIEW

**Comment by jaj22** [comment]
Were there any functional changes in civInit aside from the postMortem call?

**Comment by igorkis-scrts** [comment]
Nope, it's just a light refactoring of that particular function + postmortem call.

File: A3A/addons/core/functions/CREATE/fn_CIVinit.sqf [hunk 1]
```diff
@@ -1,83 +1,70 @@
...
+
+    if (time - (_victim getVariable ["lastInjuredByPlayer", 0]) < 120) then {
+        _killer = _victim getVariable ["injuredByPlayer", _killer];
+    };
+    if (isNull _killer) then {
+        _killer	= _victim;
+    };
+    
+    if (_victim == _killer) then {
+        _nul = [-1,-1,getPos _victim] remoteExec ["A3A_fnc_citySupportChange",2];
```
Comment (jaj22) [inline A3A/addons/core/functions/CREATE/fn_CIVinit.sqf:41]: Can you switch all these getPos to getPosATL while you're in there? citySupportChange doesn't care about the Z value.

Comment (igorkis-scrts) [reply A3A/addons/core/functions/CREATE/fn_CIVinit.sqf]: Done. 🐸

**Review by jaj22 — APPROVED** [review]

**Comment by Bob-Murphy** [comment]
@igorkis-scrts merge conflicts

**Comment by igorkis-scrts** [comment]
@Bob-Murphy 
Done.

**Pull request merged**